quickstarts/spring-camel-drools-decision-table/pom.xml

```diff
@@ -321,6 +420,18 @@
   </build>
 
   <profiles>
+       <profile>
+            <id>deploy</id>
+            <properties>
+                <deploy.skip>false</deploy.skip>
+            </properties>
+        </profile>
+       <profile>
+            <id>eap</id>
+            <properties>
+                <kie.post.processor.class>org.kie.spring.KModuleBeanFactoryPostProcessor</kie.post.processor.class>
+            </properties>
+        </profile>
```

Comment:
Nit picking, but we should use a consistent indents (4 spaces) in the XML files. This file uses few different indent levels on different lines....